packages/ui/src/app/pages/WorkingGroups/WorkingGroupsOpening.tsx

```diff
@@ -96,6 +106,16 @@ const WorkingGroupOpening = () => {
       </MainPanel>
       <ContentWithSidepanel>
         <MarkdownPreview markdown={opening.description} />
+        <SidePanel>
+          {opening.status !== OpeningStatuses.UPCOMING && (
+            <ApplicantsList
+              allApplicants={opening.applications}
+              myApplication={hiredMember?.member.id === activeMembership?.id ? activeMembership : undefined}
+              hired={hiredMember?.member}
+              hiringComplete={opening.status === OpeningStatuses.CANCELLED}
```

Comment:
```suggestion
              hiringComplete={opening.status !== OpeningStatuses.OPEN}
```

That's because:

```typescript
union WorkingGroupOpeningStatus = OpeningStatusOpen | OpeningStatusFilled | OpeningStatusCancelled
```